usaspending_api/references/management/commands/load_agencies.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand, CommandError
+from usaspending_api.references.models import Agency, SubtierAgency
```

Comment:
Ah, yeah, this didn't throw any errors because there's no test for it I believe.